WordPress/src/main/res/values/strings.xml

```diff
@@ -2397,6 +2397,7 @@
     <string name="reader_empty_search_title">No results found</string>
     <string name="reader_empty_search_description">No results found for %s for your language</string>
     <string name="reader_empty_search_request_failed">Unable to perform search</string>
+    <string name="reader_empty_subscriptions">You haven\'t subscribed to any blog yet</string>ee
```

Comment:
Update to remove erroneous trailing `ee` characters and improve clarity of the statement. 
```suggestion
    <string name="reader_empty_subscriptions">You haven\'t subscribed to any blogs yet</string>
```